src/spikeinterface/widgets/traces.py

```diff
@@ -473,11 +477,11 @@ def plot_sortingview(self, data_plot, **backend_kwargs):
         try:
             import pyvips
         except ImportError:
-            raise ImportError("To use the timeseries in sorting view you need the pyvips package.")
+            raise ImportError("To use `plot_traces()` in sortingview you need the pyvips package.")
```

Comment:
This is showing up as an unused import? Is this necessary somewhere else?

Reply:
It is used internally by the `sortingview` backend. I'd leave it here.
Yes it should be `plot_traces` :)

Reply:
Cool thanks. I've only tested `sortingview` once, so other than Jeremy's talk (& yours) I didn't know too much about it.